week7/meal-sharing-final/backend/Program.cs

```diff
@@ -0,0 +1,121 @@
+using Dapper;
+// using meal_sharing_final.meal;
+using MySql.Data.MySqlClient;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+// builder.Services.AddScoped<IMealRepository, MealRepository>();
+
+var app = builder.Build();
+
+app.MapGet("/", () => "week 7 meal sharing final!");
+
+// app.MapGet("/api/meals", () => new[] { new Meal(){ Title= "momo", MaxReservations = 111, Price=111
+
+// }});
+app.MapGet("/api/meals", async (IConfiguration configuration) =>
+{
+  // // starting from C# 8 you no longer need inner {}
+  using var connection = new MySqlConnection(configuration.GetConnectionString("Default"));
+  var meals = await connection.QueryAsync("SELECT * FROM meal");
+  return meals;
```

Comment:
This part should be in `MealRepository`/`IMealRepository`. You can create a method called `GetAll`

Reply:
done. thank you